build/version.gni

```diff
@@ -2,8 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-# This exposes the Chrome version and Crosswalk version as GN variables for
-# use in build files.
+# This exposes the Chrome version as GN variables for use in build files.
```

Comment:
I commented a while ago about how changing the existing parts of this file was not necessary, and I still do.